boa/src/syntax/lexer/regex.rs

```diff
@@ -130,19 +130,6 @@ impl<R> Tokenizer<R> for RegexLiteral {
     }
 }
 
-bitflags! {
-    /// Flags of a regular expression.
-    #[derive(Default)]
-    struct RegExpFlags: u8 {
-        const GLOBAL = 0b0000_0001;
-        const IGNORE_CASE = 0b0000_0010;
-        const MULTILINE = 0b0000_0100;
-        const DOT_ALL = 0b0000_1000;
-        const UNICODE = 0b0001_0000;
-        const STICKY = 0b0010_0000;
-    }
-}
-
 fn parse_regex_flags(s: &str, start: Position, interner: &mut Interner) -> Result<Sym, Error> {
```

Comment:
I would move this function together with the structure, and change it so that this implements the `FromStr` trait for the structure.